src/lib/multiledger.js

```diff
@@ -1,21 +1,16 @@
 'use strict'
 
 const _ = require('lodash')
-const FiveBellsLedger = require('../ledgers/five-bells-ledger')
 const healthStatus = require('../common/health.js')
 
 function Multiledger (options) {
```

Comment:
Should the connector use `ilp-core` instead of this multiledger abstraction, or is that for another PR?